java/kafka/consumer/pom.xml

```diff
@@ -52,6 +48,10 @@
             <groupId>io.strimzi</groupId>
             <artifactId>kafka-oauth-client</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+        </dependency>
```

Comment:
Why moving this dependency in the Kafka related examples? I don't think it's needed.

Reply:
Removed

Reply:
I know it's used, it's the Kafka based example but AFAICS you moved from one place to another and in a PR which is not related to the Kafka examples.

Reply:
You moved the same block from line 19 to line 51 for now reasons :-) This is the point I am making.

Reply:
Ah sorry. Now I get you. I must have deleted it from the wrong pom and then realised I had to import it again and it was placed in a different spot. That's why it showed up in the PR. Putting back to the original place.